src/app/chat/views/channels_list.nim

```diff
@@ -66,6 +86,12 @@ QtObject:
     else:
       result = idx
 
+  proc getChannelByName*(self: ChannelsList, name: string): ChatItem =
+    for chat in self.chats:
+      if chat.name == name:
+        return chat
+    raise newException(OSError, fmt"No chat found with the name {name}")
```

Comment:
Is this type of exception captured in the nim side, and logged?

Reply:
Not sure. I catch it on the other side of the call anyway